src/components/usa-details/index.js

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html, css, unsafeCSS, nothing } from "lit";
+import { classMap } from "lit/directives/class-map.js";
+import styles from "./usa-details.css.js";
+import uswdsCoreStyles from "@uswds/uswds/scss/uswds-core?inline";
+
+/**
+ * @summary The usa-details component.
+ *
+ * @slot details-body - Content for the details panel
+ *
+ * @attribute {Boolean} open - Set the panel to be open on load
+ * @attribute {String} name - Add a group name if the element is part of a details group
+ *
+ * @cssprop --theme-details-font-family - Sets the font family for the details element
+ * @cssprop --theme-details-border-color - Sets the border width for the details element
+ * @cssprop --theme-details-border-width - Sets the border color for the details element
+ * @cssprop --theme-details-background-color - Sets the background color of the content panels
+ * @cssprop --theme-details-summary-background-color - Sets the background color of the summary element
+ *
+ * @tagname usa-details
+ */
+export class UsaDetails extends LitElement {
+  static styles = [unsafeCSS(uswdsCoreStyles), styles];
+
+  static properties = {
+    bordered: { type: Boolean }
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.details = [...this.querySelectorAll('details')];
+  }
+
+  template() {
+    const classes = {
+      "usa-details": true,
+      "usa-details__bordered": this.bordered
+    }
+    return html`
+      ${this.details.map((detail) => {
+          this.summary = detail.querySelector('summary');
+          this.content = detail.querySelector('[slot="details-body"]');
+          this.open = detail.getAttribute('open');
+          this.name = detail.getAttribute('name');
+          this.summary.classList.add('usa-details__summary');
+          this.content.classList.add('usa-details__content');
+
+          return html`
+            <details class="${classMap(classes)}" open="${this.open || nothing}" name="${this.name || nothing}">
```

Comment:
**Issue:** In native disclosure, the `open` attribute doesn't have a value. For example something like `open="false"` will still keep the element open.


We should follow this pattern to keep things consistent. 

[The Details disclosure element - HTML: HyperText Markup Language | MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#open)